bundle-monix-http4s-blaze/src/main/scala/com/avast/sst/bundle/MonixServerApp.scala

```diff
@@ -0,0 +1,36 @@
+package com.avast.sst.bundle
+
+import cats.effect.{ExitCode, Resource}
+import monix.eval.{Task, TaskApp}
+import org.http4s.server.Server
+import org.slf4j.LoggerFactory
+
+/** Extend this `trait` if you want to implement server application using [[monix.eval.Task]] effect data type.
+  *
+  * Implement method `program` with initialization and business logic of your application. It will be automatically run until JVM is shut
+  * down in which case all the resources are cleaned up because the whole `program` is a [[cats.effect.Resource]].
+  */
+trait MonixServerApp extends TaskApp {
```

Comment:
It looks like this project is kind of example, right? Would be better to have it just shown in a README? It's scary how many modules we have here...

Reply:
No, this is not an example, this is a helper `trait` to simplify server initialization. And the submodule in itself "bundles" together certain dependencies which is the more important feature.

Reply:
Yes, but this requires that the program must have at least one HTTP server based on Blaze.

It would make more sense for me having a module that would contain a `MonixResourceApp` base class/trait that would have `A` instead of `org.http4s.server.Server`.

And the same for `ZioServerApp` - I would prefer to have _generic_ `ZioResourceApp` instead.

Reply:
Good point, I guess I'll add it but the problem will be that the server app automatically "blocks" whereas you don't want to do that for generic `A`. So I think we need both. I've done this one automatically because the project is called "server" so it seemed natural to provide that.